scripts/init-properties-for-docker.sh

```diff
@@ -21,7 +21,7 @@ ctia.store.es.default.host=$ip
 ctia.store.es.default.port=9200
 ctia.hook.es.host=$ip
 ctia.hook.es.port=9200
-ctia.hook.redis.uri=http://${ip}:6379
```

Comment:
The expression should be wrapped in double quotes. So if there is ever an error that puts a space into the "ip" var, you don't accidentally set the uri to only include the first thing, and then get an error as it tries to execute the next term